Javascript/script.js

```diff
@@ -0,0 +1,138 @@
+var firstnum=0,secondnum=0,operator,result=0;
+function openCalculator()
+{
+  firstnum=0,secondnum=0,operator=null,result=0;
+  calculator=document.createElement("div");
+  calculator.setAttribute("class","calculator");
+
+  displayResult=document.createElement("div");
+  displayResult.setAttribute("class","displayResult");
+  calculator.append(displayResult);
+
+  displayButton=document.createElement("div");
+  displayButton.setAttribute("class","displayButton");
+  calculator.append(displayButton);
+
+  digit7=document.createElement("button");
+  digit7.innerHTML=7;
```

Comment:
You can use a loop here to add all digit

Reply:
done with all changes

Reply:
removed eval